webrtc.html

```diff
@@ -866,7 +866,7 @@
 
                   <li>
                     <p>If <var>connection</var>'s <a>signaling state</a>
-                    changed above, fire a simple event named <code><a href=
+                    changed above, queue a task to fire a simple event named <code><a href=
```

Comment:
If we want to queue a new task here I think we want to do the state update in that task as well. But I didn't really understand from @jan-ivar if it was necessary. What do you say @jan-ivar?